### DISCUSSION TITLE
Deployment help

### DISCUSSION THREAD

**Discussion by Lakshya0507** [vercel/community#7259] [Help]
### Hello,
I have a _problem_ with deploment can check my app and tell me to start the app I use the command `npm run dev`. I am giving a link to the github repo: [https://github.com/Lakshya0507/My-APP.git](https://github.com/Lakshya0507/My-APP.git)

### Steps to Reproduce

`> Build error occurred
Error: > Couldn't find a `pages` directory. Please create one under the project root
    at Object.findPagesDir (/vercel/path0/node_modules/next/dist/lib/find-pages-dir.js:39:15)
    at /vercel/path0/node_modules/next/dist/build/index.js:94:70
    at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
    at async Object.build [as default] (/vercel/path0/node_modules/next/dist/build/index.js:64:29)
Error: Command "npm run build" exited with 1
`

**Comment by versecafe** [comment]
@Lakshya0507 You're missing some necessary files a `src/app/layout.tsx` & `src/app/page.tsx` for app router, you can also remove the `src` dir if desired. If it works in local development you may just have untracked files so run `git add -A` & `git commit -a -m "missing files"`

[a comment by an automated account is not shown]
